src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/config/ControllerAdvice.kt

```diff
@@ -52,7 +53,21 @@ class ControllerAdvice {
 
   @ExceptionHandler(RestClientResponseException::class)
   fun handleRestClientException(e: RestClientResponseException): ResponseEntity<ErrorResponse> {
-    log.error("RestClientResponseException: {}", e.message)
+    log.error("RestClientResponseException: {}", e.message, e)
+    return ResponseEntity
+      .status(e.statusCode)
+      .body(
+        ErrorResponse(
+          status = e.statusCode.value(),
+          userMessage = "Rest client exception ${e.message}",
+          developerMessage = e.message,
+        ),
+      )
+  }
+
+  @ExceptionHandler(WebClientResponseException::class)
```

Comment:
these weren't being handled before - this was being logged as a weird reactor error